platform/projects_and_organizations/organization_settings.mdx

```diff
@@ -10,13 +10,18 @@ import { ScreenshotWrapper } from '../shared/_ScreenshotWrapper';
 
 This page shows the **list of all members in your organization**.
 
-You can invite new members by clicking the **Invite user** button. Then **enter their email address or create an invitation link** to share with them.
+You can invite new members by clicking the **Invite user** button in the upper right corner. 
+Use the toggle on the right-hand side to choose between **entering their email address or creating an invitation link**, to share with them.
 
-You can **set the role of the member**. The role can be one of the following:
+Before sending an invite **set the role of a member**. The role can be one of the following:
 
 - **Member**: The member has basic access to all organization projects.
-- **Maintainer**: Same as the member, but also can create new projects and create/edit glossaries.
-- **Owner**: The owner of the organization. The owner can manage the organization settings, manage the glossaries, manage the projects, and manage the members.
+- **Maintainer**: Same as the member, additionally can create new projects and create/edit glossaries.
+- **Owner**: The owner of the organization. The owner can manage: the organization settings, the glossaries, the projects and the members.
+
+:::tip

```

Comment:
I'd remove this tip completely, as to me it's not really a tip and the user can see the the next chapter right below.